packages/ui-extensions/docs/surfaces/checkout/build-docs.sh

```diff
@@ -19,19 +18,23 @@ run_sed() {
   fi
 }
 
+if [ "$API_VERSION" = "unstable" ]; then
+  echo "Publishing docs for the 'unstable' checkout UI extensions API is not supported. You must specify a calver version YYYY-MM or YYYY-MM-rc (for a release candidate)."
+  exit 1;
+fi
+
 if [ -z $API_VERSION ]
 then
-  API_VERSION="unstable"
-  echo "Building docs for 'unstable' checkout UI extensions API. You can add a calver version argument (e.g. 'yarn docs:checkout 2023-07') to generate the docs for a stable version."
+  echo "You must specify a calver version YYYY-MM or YYYY-MM-rc (for a release candidate)."
+  exit 1;
 else
   echo "Building docs for '$API_VERSION' checkout UI extensions API."
-  echo "When generating docs for a stable version, 'unstable' docs are not regenerated. This avoids overwriting other unstable changes that are not included in this version."
-  echo "If you need to update the 'unstable' version, run this command again without the '$API_VERSION' parameter."
 fi
 
-COMPILE_DOCS="yarn tsc --project $DOCS_PATH/tsconfig.docs.json --types react --moduleResolution node  --target esNext  --module CommonJS && yarn generate-docs --overridePath ./$DOCS_PATH/typeOverride.json --input ./$DOCS_PATH/reference ./$SRC_PATH --typesInput ./$SRC_PATH ../ui-extensions-react/$SRC_PATH --output ./$DOCS_PATH/generated"
+COMPILE_DOCS="yarn tsc --project $DOCS_PATH/tsconfig.docs.json --types react --moduleResolution node  --target esNext  --module CommonJS && yarn generate-docs --overridePath ./$DOCS_PATH/typeOverride.json --input ./$DOCS_PATH/reference ./$SRC_PATH --typesInput ./$SRC_PATH  --output ./$DOCS_PATH/generated"
```

Comment:
Doesn't this also need the preact path?

```suggestion
COMPILE_DOCS="yarn tsc --project $DOCS_PATH/tsconfig.docs.json --types react --moduleResolution node  --target esNext  --module CommonJS && yarn generate-docs --overridePath ./$DOCS_PATH/typeOverride.json --input ./$DOCS_PATH/reference ./$SRC_PATH --typesInput ./$SRC_PATH ./$SRC_PATH/preact/  --output ./$DOCS_PATH/generated"
```

Because that's a separate [entry point](https://github.com/Shopify/ui-extensions/blob/2025-07-rc/packages/ui-extensions/loom.config.ts#L22). None of the hooks are exported from [$SRC_PATH](https://github.com/Shopify/ui-extensions/blob/2025-07-rc/packages/ui-extensions/src/surfaces/checkout.ts)

Reply:
The doc gen tool extracts types by traversing the `[$SRC_PATH](https://github.com/Shopify/ui-extensions/blob/2025-07-rc/packages/ui-extensions/src/surfaces/checkout)` rather than exports from a module. So, since the preact hooks are located within `/src/surfaces/checkout`, they are automatically picked up.